crawler.py

```diff
@@ -90,57 +90,11 @@ def test_crash(driver):
 var crash = badptr.contents;""")
 
 
-def get_domain_list(logger, n_sites, out_path):
-    """Load the top million domains from disk or the web"""
-    top_1m_file = os.path.join(out_path, MAJESTIC_URL.split('/')[-1])
-    pyfunc_cache_file = os.path.join(out_path, 'pyfunceable_cache.json')
-
-    # download the file if it doesn't exist or if it's more than a week stale
-    if (not os.path.exists(top_1m_file) or
-            time.time() - os.path.getmtime(top_1m_file) > WEEK_IN_SECONDS):
-        logger.info(
-            "Loading new Majestic data and refreshing PyFunceble cache")
-        response = urlopen(MAJESTIC_URL)
-        with open(top_1m_file, 'w') as f:
-            f.write(response.read().decode())
-
-        # if the majestic file is expired, let's refresh the pyfunceable cache
-        if os.path.exists(pyfunc_cache_file):
-            os.remove(pyfunc_cache_file)
-
-    # load cache
-    if os.path.exists(pyfunc_cache_file):
-        with open(pyfunc_cache_file) as f:
-            pyfunc_cache = json.load(f)
-    else:
-        pyfunc_cache = {}
-
-    domains = []
-    with open(top_1m_file) as f:
-        # first line is CSV header
-        next(f)
-
-        # only read the first n_sites lines
-        for l in f:
-            domain = l.split(',')[2]
-
-            if domain in pyfunc_cache:
-                if pyfunc_cache[domain] == 'ACTIVE':
-                    domains.append(domain)
-            else:
-                status = PyFunceble(domain, config=OUR_PYFUNCEBLE_CONFIG)
-                logger.info('PyFunceble: %s is %s', domain, status)
-                if status == 'ACTIVE':
-                    domains.append(domain)
-                pyfunc_cache[domain] = status
-
-            if len(domains) >= n_sites:
-                break
-
-    # save pyfunceble cache again
-    with open(pyfunc_cache_file, 'w') as f:
-        json.dump(pyfunc_cache, f)
-
+def get_domain_list(n_sites):
```

Comment:
The signature of `get_domain_list()` changed, but none of the callers use the updated signature.

Reply:
Good catch, thank you!